backend/supervisord.conf

```diff
@@ -33,9 +33,9 @@ stopasgroup=true
 # since this is often the bottleneck for "sync" jobs (e.g. document set syncing, 
 # user group syncing, deletion, etc.) 
 [program:celery_worker_light]
-command=bash -c "celery -A danswer.background.celery.celery_run:celery_app worker \
+command=bash -c "celery -A danswer.background.celery.versioned_apps.light worker \
     --pool=threads \
-    --concurrency=${CELERY_WORKER_LIGHT_CONCURRENCY:-24} \
+    --concurrency=${CELERY_WORKER_LIGHT_CONCURRENCY:-${NUM_INDEXING_WORKERS:-24}} \
```

Comment:
small merge issue, `NUM_INDEXING_WORKERS` should not be here.

Also, can we move all of these configs over to the config files directly?